### PR TITLE
Fix Cache for multiple Loaders

### PR DESCRIPTION
**Pull request by maennchen**

### CODE REVIEW

File: src/loader.ts [hunk 1]
```diff
@@ -26,7 +26,7 @@ interface CachedSchema {
   schema: GraphQLSchema;
 }
 
-let cachedSchema: CachedSchema | null = null;
+let cachedSchemas: { [s: string]: CachedSchema; } = {}
```
Comment (stephen) [inline src/loader.ts]: consider `Record<string, CachedSchema>`

**Comment by jeremyzahner** [comment]
@stephen Any chance of getting this released? =)

**Comment by stephen** [comment]
@jeremyzahner released in 0.1.1!

**Comment by jeremyzahner** [comment]
@stephen Nice, thanks.

**Pull request merged**